### PR TITLE
feat(pricing): add Team Tiers

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- feat(pricing): add Team Tiers

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: src/components/Pricing.tsx [hunk 1]
```diff
@@ -169,21 +46,261 @@ export const Pricing = () => {
...
+      amount: '$300',
+      quantity: '1M',
+    },
+    {
+      amount: '$700',
+      quantity: '2.5M',
+    },
+    {
+      amount: '$1,250',
+      quantity: '$5M',
```
Comment (eseidel) [inline src/components/Pricing.tsx:147]:
```suggestion
      quantity: '5M',
```

File: src/components/Pricing.tsx [hunk 2]
```diff
@@ -169,21 +46,261 @@ export const Pricing = () => {
...
+    {
+      amount: '$2,000',
+      quantity: '10M',
+    },
+  ];
+
+  const features: Feature[] = [
+    { title: 'Unlimited apps' },
+    { title: 'Unlimited developers' },
+    { title: 'Discord & email support' },
```
Comment (eseidel) [inline src/components/Pricing.tsx:158]: I'd like this to change to private support beyond 5M?

File: src/components/Pricing.tsx [hunk 3]
```diff
@@ -169,21 +46,261 @@ export const Pricing = () => {
...
+          <p className="mb-2 text-xl font-bold font-heading text-white text-left">
+            Enterprise
+          </p>
+          <div className="flex justify-start items-end">
+            <div className="text-3xl sm:text-4xl font-bold text-white text-left mt-4 mr-2 mb-2">
+              Custom
+            </div>
+          </div>
+          <p className="mt-4 mb-4 text-gray-500 leading-loose text-left">
+            For 10M+ user apps.
```
Comment (eseidel) [inline src/components/Pricing.tsx:285]:
```suggestion
            For very large apps.
```

**Review by eseidel — APPROVED** [review]

File: src/components/Pricing.tsx (hunk 2 above)
Comment (felangel) [reply src/components/Pricing.tsx:158]: sounds good

**Pull request merged**